### PR TITLE
Animation: Circle Alignment on Bounce Animation part

### LINKED ISSUE #3115: Scale Anchor Misalignment in Bounce & Zoom Animation Parts
Some items that are meant to scale together are slightly misaligned when animating in. May be a precision error or transform-origin not being applied properly.

Primarily seen with link dots. Visible in Fashion & Fitness template on link pages.

https://github.com/google/web-stories-wp/pull/3112#issuecomment-656321512

### PR DESCRIPTION
**Pull request by maxyinger**
## Summary
This fixes the alignment issue seen here in the fitness template while the dots where animating:

093f3e0f-b2aa-459b-82d5-587dbb679a72

## Relevant Technical Choices
Turns out this is a chrome problem, I'm guessing it's an issue of chrome internally rounding off a needed level of precision when scaling clip paths that need super high precision like this one. You can see the issue highlighted here where I've applied a large scale transform to the circle element. The element itself is scaling properly, but the calculation for the clip-path is not:

Screen Shot 2020-08-18 at 11 39 48 AM

Being an internal chrome problem, the only solution was finding a workaround. To accomplish this, I made a bullseye shape for the masks panel that combines the outer and inner circle. This solution wasn't affected by chrome's precision issues for scaling. It also provides the user a better experience regardless of animations cus they can select the single bullseye object and alter the color.

## To-do
NA

## User-facing changes
Bullseye now one shape in Fitness template.

## Testing Instructions
Go to Templates -> Fitness -> Create Story from template -> navigate to page 7 where these bullseyes appear & change the color of one. Color should change the entire bullseye (both dot and circle).

## Screenshots
**(Bouncing Bullseyes don't misalign)**
bounce_alignment_bug

---

Fixes #3115

### CODE REVIEW

File: assets/src/dashboard/templates/raw/fitness.json [hunk 1]
```diff
@@ -838,7 +838,7 @@
           "resource": {
             "type": "image",
             "mimeType": "image/jpeg",
-            "src": "http://localhost:8899/wp-content/plugins/web-stories/assets/images/templates/fitness/fitness_page4_bg.jpg",
+            "src": "http://localhost:8899/wp-content/plugins/web-stories//assets/images/templates/fitness/fitness_page4_bg.jpg",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/fitness.json]: There are now double slashes everywhere 🤔

File: assets/src/dashboard/templates/raw/fitness.json [hunk 2]
```diff
@@ -2541,7 +2469,7 @@
           "type": "text",
           "basedOn": "96fea994-b71e-442f-b611-16d73f520016",
           "id": "9b117b87-0269-4c4b-9e0d-8cdbcb0df89a",
-          "content": "<span style=\"color: rgba(255, 255, 255, 1)\"><span style=\"font-weight: 600\">10 BEST CORETRAINING EXERCISES</span></span>",
+          "content": "<span style=\"color: rgba(255, 255, 255, 1)\"><span style=\"font-weight: 600\">10 BEST CORE TRAINING EXERCISES</span></span>",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/fitness.json]: Looks like there are now some invisible characters between `core` and `training`. Please revert this change.

File: assets/src/dashboard/templates/raw/fitness.json [hunk 3]
```diff
@@ -2492,7 +2420,7 @@
           "type": "text",
           "basedOn": "0884aabc-ca1e-4b37-a87b-68a6fbf377db",
           "id": "96fea994-b71e-442f-b611-16d73f520016",
-          "content": "<span style=\"font-weight: 600; color: #fff\">SWEAT</span>\n<span style=\"font-weight: 600; color: #fff\">RESISTENT</span>\n<span style=\"font-weight: 600; color: #fff\">EARBUDS</span>",
+          "content": "<span style=\"font-weight: 600; color: #fff\">SWEAT</span>\n<span style=\"font-weight: 600; color: #fff\">RESISTENT</span>\n<span style=\"font-weight: 600; color: #fff\"> EARBUDS</span>",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/fitness.json]: What change was made here? Seems like there might be some invisible chars too.

Comment (maxyinger) [reply assets/src/dashboard/templates/raw/fitness.json]: Yea I noticed these in the diff too, but nothing changed here. Just deleted the two circles on page 7 and replace them with the new bullseye shape

Comment (swissspidy) [reply assets/src/dashboard/templates/raw/fitness.json]: I see a `U+2028` character (line separator) being added here when checking via https://www.soscisurvey.de/tools/view-chars.php. Maybe it's an OS thing, not sure 🤷 

I remember this caused issues before, see https://github.com/google/web-stories-wp/issues/2473#issuecomment-651509687

Comment (swissspidy) [reply assets/src/dashboard/templates/raw/fitness.json]: In 9b0e5b3b234fdf8c84959f365795746cf1c230b8 I've added a quick test for this. Hope that's OK. Reverting the unintended changes here in this file should make it pass.

Comment (maxyinger) [reply assets/src/dashboard/templates/raw/fitness.json]: yea absolutely ok. sweet. thanks.

Comment (swissspidy) [reply assets/src/dashboard/templates/raw/fitness.json]: Cool. In the future this is could be moved to a linter, bit for now it should work.

**Review by mariano-formidable — APPROVED** [review]
Looks good to me!  Nice bullseye shape!
